Makefile

```diff
@@ -24,6 +25,12 @@ opam-create-switch: ## Create opam switch
 opam-install: ## Install development dependencies
 	cd ocaml-tree && npm install
 	opam install -y ocaml-lsp-server
+	env IGNORECONSTRAINTS="melange,mel"
+	opam pin add dune https://github.com/ocaml/dune.git#0d44bbfdb2a68907a464aeb2dabe95388dac5712 -y
+	opam pin add melange-compiler-libs --dev-repo -y
+	opam install luv reason -y
+	opam pin add melange . --with-test -y
+	opam pin add mel . --with-test -y
```

Comment:
Same about these two (we just need to pin the dependencies of melange packages, not the melange packages themselves).